lib/install/management/delete.go

```diff
@@ -46,10 +46,28 @@ func (d *Dispatcher) DeleteVCH(conf *config.VirtualContainerHostConfigSpec) erro
 		return nil
 	}
 
-	if err = d.DeleteVCHInstances(vmm, conf); err != nil {
-		// if container delete failed, do not remove anything else
-		log.Infof("Specify --force to force delete")
-		return err
+	d.parentResourcepool, err = d.getComputeResource(vmm, conf)
+	if err != nil {
+		log.Error(err)
+		if !d.force {
+			log.Infof("Specify --force to force delete")
+			return err
+		}
+		// Can't find the RP VCH was created in to delete cVMs, continue anyway
+		log.Warnf("No container VMs found, but proceeding with delete of VCH due to --force")
+		err = nil
```

Comment:
getComputeResource never returns non-nil for both rp and err. If you set `err = nil` here then we will still skip over the delete of the containerVMs (not sure why the method is called `DeleteVCHInstances`).

I don't know what that will mean for deleting images if they are still attached to cVMs - I suspect it will fail with errors about locked vmdks if any of the cVMs are powered on. I don't see that that will cause any other "interesting" behaviours like deleting non-VCH related things, but please sanity check me if you haven't looked at this path.